src/main/java/Domain/EntityManager.java

```diff
@@ -65,11 +65,85 @@ public static EntityManager getInstance() {
             admin.addNewRole(new SystemAdmin(admin));
             admin.addNewRole(new AssociationRepresentative(admin));
             admin.addNewRole(new Referee(admin,RefereeQualification.VAR_REFEREE));
+            admin.addNewRole(new TeamOwner(admin));
+            createObjectsForGuiTests();
         }
 
         return entityManagerInstance;
     }
 
+    private static void createObjectsForGuiTests() {
```

Comment:
and this 😮

Reply:
Without them, the tests would not pass

Reply:
אבל אז האובייקטים האלה יהיו חלק מהמערכת הסופית לא? 
אנחנו רוצים שהם יווצרו רק לצרכי טסטים ושהמשתמשים הרגילים לא יראו אותם

Reply:
i agree but that should not be present in the master...

Reply:
We will delete this data after we add the DB